### PR TITLE
Intel.ProcessorDiagnosticTool64bit version 4.1.5.37

### PR DESCRIPTION
**Pull request by Rolling2405**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26676)

### CODE REVIEW

**Comment by Rolling2405** [comment]
Submission containing materials of a third party: Intel Corporation

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
+PackageVersion: 4.1.5.37
```
Comment (OfficialEsco) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.installer.yaml:5]:
```suggestion
PackageVersion: 4.1.5
```

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
+PackageVersion: 4.1.5.37
```
Comment (OfficialEsco) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.1.5
```

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
+PackageVersion: 4.1.5.37
```
Comment (OfficialEsco) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.yaml:5]:
```suggestion
PackageVersion: 4.1.5
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
```
Comment (ItzLevvie) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.installer.yaml:4]:
```suggestion
PackageIdentifier: Intel.ProcessorDiagnosticTool
```

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
```
Comment (ItzLevvie) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Intel.ProcessorDiagnosticTool
```

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
```
Comment (ItzLevvie) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.yaml:4]:
```suggestion
PackageIdentifier: Intel.ProcessorDiagnosticTool
```

File: manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool64bit
+PackageVersion: 4.1.5.37
+PackageLocale: en-US
+Publisher: Intel Corporation
+PackageName: Intel Processor Diagnostic Tool 64bit
+License: Intel Corporation
+ShortDescription: Intel Processor Diagnostic and Stress Tool
```
Comment (ItzLevvie) [inline manifests/i/Intel/ProcessorDiagnosticTool64bit/4.1.5.37/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml:10]:
```suggestion
ShortDescription: The purpose of the Intel® Processor Diagnostic Tool is to verify the functionality of an Intel® microprocessor.
Description: The purpose of the Intel® Processor Diagnostic Tool is to verify the functionality of an Intel® microprocessor. The diagnostic tool checks for brand identification, verifies the processor operating frequency, tests specific processor features, and performs a stress test on the processor.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
```
 Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\i\Intel\ProcessorDiagnosticTool64bit\4.1.5.37', 
expected 'manifests\i\Intel\ProcessorDiagnosticTool\4.1.5'
```

**Comment by Rolling2405** [comment]
> ```
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
> actual 'manifests\i\Intel\ProcessorDiagnosticTool64bit\4.1.5.37', 
> expected 'manifests\i\Intel\ProcessorDiagnosticTool\4.1.5'
> ```

How do I resubmit my manifest to fix this without creating a new submission?

**Comment by OfficialEsco** [comment]
> How do I resubmit my manifest to fix this without creating a new submission?

You use GitHub Desktop to rename the Software folder.

**Comment by quhxl** [comment]
You also have to rename the files.

**Comment by Rolling2405** [comment]
> You also have to rename the files.

What do you mean??

**Comment by OfficialEsco** [comment]
Open file explorer and go to the forked repo, rename 
```
/manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool64bit.installer.yaml
/manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml
/manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool64bit.yaml
```
to
```
/manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml
/manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.locale.en-US.yaml
/manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.yaml
```

**Comment by Rolling2405** [comment]
> Open file explorer and go to the forked repo, rename
> 
> ```
> /manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool64bit.installer.yaml
> /manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool64bit.locale.en-US.yaml
> /manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool64bit.yaml
> ```
> 
> to
> 
> ```
> /manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml
> /manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.locale.en-US.yaml
> /manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.yaml
> ``
Thanks figured out by looking at some examples! Thanks anyways!

File: manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool
+PackageVersion: 4.1.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://downloadmirror.intel.com/19792/eng/ipdt_installer_4.1.5.37_64bit.msi
+  InstallerSha256: FAFE723C3FF07086183693BEE910F8240AED9B1B6698008C0410DBC2F95CCAE4
+  ProductCode: '{6E05E656-6ED8-49DE-AA9C-C4677F7086C5}'
```
Comment (quhxl) [inline manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml:12]:
```suggestion
  ProductCode: '{6E05E656-6ED8-49DE-AA9C-C4677F7086C5}'
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: msi
  InstallerUrl: https://downloadmirror.intel.com/19792/eng/ipdt_installer_4.1.5.37_32bit.msi
  InstallerSha256: 05168596E7B5F7F5778125858454C0E4BB7A04997F67274831C7F5F24D058C6C
  ProductCode: '{57DD224D-404E-49FE-AB9A-A77A7C3CB02D}'
```

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml (hunk 8 above)
Comment (Rolling2405) [reply manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml:12]: Isn't 64 bit better than 32 bit?

Comment (Rolling2405) [reply manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml:12]: > ```suggestion
> 
>   ProductCode: '{6E05E656-6ED8-49DE-AA9C-C4677F7086C5}'
> 
> - InstallerLocale: en-US
> 
>   Architecture: x86
> 
>   InstallerType: msi
> 
>   InstallerUrl: https://downloadmirror.intel.com/19792/eng/ipdt_installer_4.1.5.37_32bit.msi
> 
>   InstallerSha256: 05168596E7B5F7F5778125858454C0E4BB7A04997F67274831C7F5F24D058C6C
> 
>   ProductCode: '{57DD224D-404E-49FE-AB9A-A77A7C3CB02D}'
> 
> ```

Isn't 64 bit better than 32 bit?

Comment (OfficialEsco) [reply manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.installer.yaml:12]: This suggestion adds the x32 for compatability

**Comment by Rolling2405** [comment]
@OfficialEsco Can you please review this submission?

File: manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Intel.ProcessorDiagnosticTool
+PackageVersion: 4.1.5
+PackageLocale: en-US
+Publisher: Intel Corporation
+PackageName: Intel Processor Diagnostic Tool 64bit
```
Comment (ItzLevvie) [inline manifests/i/Intel/ProcessorDiagnosticTool/4.1.5/Intel.ProcessorDiagnosticTool.locale.en-US.yaml:8]:
```suggestion
PackageName: Intel Processor Diagnostic Tool
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Rolling2405** [comment]
@jamespik Can you check m=this submission to see if it can be merged?

**Review by OfficialEsco — APPROVED** [review]

**Comment by Rolling2405** [comment]
@OfficialEsco Can you assign someone to review it for approval so it can be merged? Currently there is no one assigned.

**Review by OfficialEsco — APPROVED** [review]
Its supposed to be merged, but got stuck

**Comment by Rolling2405** [comment]
> Its supposed to be merged, but got stuck

What does that mean? Do I need to do anything?

**Comment by OfficialEsco** [comment]
No. When we approve its supposed to get the `Moderator-Approved` label, however its not working, guess Microsoft broke the bot before the weekend

**Comment by Rolling2405** [comment]
> No. When we approve its supposed to get the `Moderator-Approved` label, however its not working, guess Microsoft broke the bot before the weekend

Got it, hopefully they fix the issue soon!

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**